data/gui/themes/default/dialogs/units_dialog.cfg

```
@@ -527,6 +558,17 @@
 									label = _ "Rename"
 								[/button]
 							[/column]
+
+							[column]
+								grow_factor = 1
+								border = "all"
+								border_size = 5
+								[button]
+									id = "mark_favorite"
+									definition = "default"
+									label = _ "Mark Favorite"
```

```suggestion
									label = _ "Favorite"
```
Honestly we could also just use the star icon for this button too, no text. People understand what a star means.

---

Used the star character.